pkg/linear-pools/contracts/midas-linear-pool/MidasLinearPoolRebalancer.sol

```diff
@@ -37,6 +38,7 @@ contract MidasLinearPoolRebalancer is LinearPoolRebalancer {
     constructor(IVault vault, IBalancerQueries queries)
         LinearPoolRebalancer(ILinearPool(ILastCreatedPoolFactory(msg.sender).getLastCreatedPool()), vault, queries)
     {
+        // solhint-disable-previous-line no-empty-blocks
         ILinearPool pool = ILinearPool(ILastCreatedPoolFactory(msg.sender).getLastCreatedPool());
         ERC20 mainToken = ERC20(address(pool.getMainToken()));
         ERC20 wrappedToken = ERC20(address(pool.getWrappedToken()));
```

Comment:
Think about maybe passing the wrapped token address in as a constructor arg then accessing its underlying tokens and decimals directly without having to make additional external calls like getLastCreatedPool(), getMainToken(), and getWrappedToken(). 

Here is a similar example of how I do it within the [SiloLinearPoolRebalancer](https://github.com/orbcollective/linear-pools/blob/silo_fix/pkg/linear-pools/contracts/silo-linear-pool/SiloLinearPoolRebalancer.sol#LL41C25-L41C25) and the [SiloLinearPoolFactory](https://github.com/orbcollective/linear-pools/blob/silo_fix/pkg/linear-pools/contracts/silo-linear-pool/SiloLinearPoolFactory.sol#L119).

Reply:
Thanks, addressed.

Reply:
I reworked this again  as in: https://github.com/orbcollective/linear-pools/pull/26#discussion_r1157831462
@baileyspraggins closing this conversation.